README.md

```diff
@@ -0,0 +1,103 @@
+# Fortify Scan on Pull Request Integration
+
+## Overview
+Please see [cdcent/ocio-fortify
+README](https://github.com/cdcent/ocio-fortify) for more information
+on what Fortify is, and how to scan your source code manually.
+
+This repository acts as a reference example for integrating your
+CDCGov repository with Fortify Code Scanning on pull requests. This
+functionality exists as two distinct loosely-coupled parts: a workflow
+associated with a Github Action Runner that contains the Fortify
+scanning utilities, and the workflow contained in this reference example. 
+
+CDCGov repositories are designed to be public-facing and open source
+by default; it would be irresponsible and insecure to expose a
+CDC-hosted Github Action Runner directly to a CDCGov repository. For
+this reason, we have created a repository on the CDCEnt organization
+to hold the proprietary runtime, and accept specific structured calls
+from external repositories. 
+
+The workflow on the CDCEnt side uses a Github Application to
+communicate with your pull request in your CDCGov repository you are
+integrating with Fortify. This app is called `fortify-pr-scan`
+[https://github.com/apps/fortify-pr-scan](https://github.com/apps/fortify-pr-scan). It
+should be enabled by default for all repositories in the CDCGov
+organization. The private key will need to be stored as a Github
+secret in your repository; it can be found here #TODO.
+
+## Prerequisites
+
+### Setting up SWA / Fortify server
+
+TODO - i don't know how to do this yet; not in EDSO-525 scope
+
+### Fortify App Version ID
+
+TODO better documentation
+
+Fortify scanning comes with a unique Fortify App Version ID. Steven
+can probably explain it better than me; but you will need this.
```

Comment:
reference this can be found in the SWA application.

Reply:
removed TODO and added better documentation